src/components/Home.vue

```diff
@@ -0,0 +1,179 @@
+<template>
+  <div class="nav">
+    <div class="left_page">
+      <div class="left-feature">
+        <div class="search">
+          <input class="search-button" type="text" placeholder="Search" />
+        </div>
+
+        <list-feature :lists="lists"></list-feature>
```

Comment:
```suggestion
        <ListFeature :lists="lists" />
```

See:

- [Component name casing in templates](https://vuejs.org/v2/style-guide/#Component-name-casing-in-templates-strongly-recommended)
- [Self-closing components](https://vuejs.org/v2/style-guide/#Self-closing-components-strongly-recommended)

Should check for other similar components.